src/pages/home/report/ReportActionCompose.js

```diff
@@ -676,17 +869,30 @@ class ReportActionCompose extends React.Component {
                                             style={[styles.textInputCompose, this.props.isComposerFullSize ? styles.textInputFullCompose : styles.flex4]}
                                             maxLines={this.state.maxLines}
                                             onFocus={() => this.setIsFocused(true)}
-                                            onBlur={() => this.setIsFocused(false)}
+                                            onBlur={() => this.setIsBlured(false)}
                                             onPasteFile={displayFileInModal}
                                             shouldClear={this.state.textInputShouldClear}
                                             onClear={() => this.setTextInputShouldClear(false)}
-                                            isDisabled={isComposeDisabled || isBlockedFromConcierge || this.props.disabled}
+                                            isDisabled={isComposeDisabled || isBlockedFromConcierge}
                                             selection={this.state.selection}
                                             onSelectionChange={this.onSelectionChange}
                                             isFullComposerAvailable={this.state.isFullComposerAvailable}
                                             setIsFullComposerAvailable={this.setIsFullComposerAvailable}
                                             isComposerFullSize={this.props.isComposerFullSize}
+                                            isSmallScreenWidth={this.props.isSmallScreenWidth}
                                             value={this.state.value}
+                                            onLayout={({
+                                                nativeEvent: {
+                                                    layout: {height},
+                                                },
+                                            }) => {
+                                                this.setState({composerHeight: height});
+                                            }}
+                                            onScroll={() => {
```

Comment:
NAB but I feel like one line would be more consistent with the rest of this file:

```js
onScroll={() => this.setState({shouldShowSuggestionMenu: false})}
```